openapi-cli/src/main/java/io/ballerina/openapi/cmd/CmdConstants.java

```diff
@@ -201,5 +201,7 @@ public String getValue() {
     // OS specific line separator
     public static final String LINE_SEPARATOR = System.lineSeparator();
     public static final String DOUBLE_LINE_SEPARATOR = LINE_SEPARATOR + LINE_SEPARATOR;
+    public static final String OPENAPI_3_1_VERSION = "V31";
+    public static final String UNSUPPORTED_OAS_3_1_ERROR = "OpenAPI definitions with version 3.1.0 are not supported";
```

Comment:
How about the below error message? 
```suggestion
    public static final String UNSUPPORTED_OAS_3_1_ERROR = "OpenAPI definitions with version 3.1.x are currently unsupported";
```

Reply:
Updated